docs/validators.md

```diff
@@ -119,18 +115,66 @@ The `validateSignature` function is used to check if a signature is valid.
 import { validateSignature } from '@taquito/utils';
 
 //valid
-const signature =
-  'edsigtkpiSSschcaCt9pUVrpNPf7TTcgvgDEDD6NCEHMy8NNQJCGnMfLZzYoQj74yLjo9wx6MPVV29CvVzgi7qEcEUok3k7AuMg';
+const signature = 'edsigtkpiSSschcaCt9pUVrpNPf7TTcgvgDEDD6NCEHMy8NNQJCGnMfLZzYoQj74yLjo9wx6MPVV29CvVzgi7qEcEUok3k7AuMg';
 const validation = validateSignature(signature);
 println(`Calling the validateSignature function with ${signature} returns ${validation}.`);
 
 //invalid checksum
-const invalidSignature =
-  'edsigtkpiSSschcaCt9pUVrpNPf7TTcgvgDEDD6NCEHMy8NNQJCGnMfLZzYoQj74yLjo9wx6MPVV29CvVzgi7qEcEUok3k7AuM';
+const invalidSignature = 'edsigtkpiSSschcaCt9pUVrpNPf7TTcgvgDEDD6NCEHMy8NNQJCGnMfLZzYoQj74yLjo9wx6MPVV29CvVzgi7qEcEUok3k7AuM';
 const invalidValidation = validateSignature(invalidSignature);
-println(
-  `Calling the validateSignature function with ${invalidSignature} returns ${invalidValidation}.`
-);
+println(`Calling the validateSignature function with ${invalidSignature} returns ${invalidValidation}.`);
+```
+
+### Validate a Block Hash
+
+The `validateBlock` function is used to check whether a block hash is valid.
+
+```js live noInline
+import { validateBlock } from '@taquito/utils';
+
+//valid
+const block ='BLJjnzaPtSsxykZ9pLTFLSfsKuiN3z7SjSPDPWwbE4Q68u5EpBw';
+const validation = validateBlock(block);
+println(`Calling the validateBlock function with ${block} returns ${validation}.`);
+
+//invalid checksum
+const invalidBlock ='BMEdgRZbJJrtByoA5Jyuvy8mzp8mefbcrno82nQCAEbBCUhog';
+const invalidValidation = validateBlock(invalidBlock);
+println(`Calling the validateSignature function with ${invalidBlock} returns ${invalidValidation}.`);
+```
+### Validate an Operation Hash
+
+The `validateBlock` function is used to check whether a block hash is valid.
```

Comment:
should be `validateOperation` function is used to check whether an operation hash is valid